Fix:
tkevent.c:24:10: fatal error: 'tclxtend.h' file not found
#include "tclxtend.h"
         ^~~~~~~~~~~~
--- src/tk/makefile.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tk/makefile	2022-05-16 21:45:47.000000000 -0500
@@ -29,10 +29,11 @@
 
 CC		= gcc
 
-CFLAGS		= -I. -I$(XINCLUDE) -I$(TCL_DIR) -O3 -DTK_VERSION=\"2.3\" -DUSE_XPM3 -DIS_LINUX
-#CFLAGS		= -I. -I$(XINCLUDE) -I$(TCL_DIR) -g -DTK_VERSION=\"2.3\" -DUSE_XPM3 -DIS_LINUX
+CFLAGS		= -I. -I$(XINCLUDE) -I$(TCL_DIR) -I$(TCLX_DIR) -O3 -DTK_VERSION=\"2.3\" -DUSE_XPM3 -DIS_LINUX
+#CFLAGS		= -I. -I$(XINCLUDE) -I$(TCL_DIR) -I$(TCLX_DIR) -g -DTK_VERSION=\"2.3\" -DUSE_XPM3 -DIS_LINUX
 
 TCL_DIR		= ../tcl
+TCLX_DIR	= ../tclx/src
 XINCLUDE	= /usr/include/X11
 XLIB		= -L/usr/X11R6/lib -lX11 -lXpm
 
